cli_tools/gce_image_publish/main.go

```diff
@@ -303,7 +249,7 @@ func populateSteps(w *daisy.Workflow, prefix string, createImages *daisy.CreateI
 	return nil
 }
 
-func (p *publish) createPrintOut(createImages *daisy.CreateImages) {
+func (p *publish.Publish) createPrintOut(createImages *daisy.CreateImages) {
```

Comment:
Any reason why Publish methods are not in the same package with the struct?

Reply:
yeah I'm debating - I want to shovel most of main.go into publish, I guess I can just also do within this PR.